libyara/grammar.y

```diff
@@ -2279,6 +2279,40 @@ rule_enumeration_item
 for_expression
     : primary_expression
       {
+        if ($1.type == EXPRESSION_TYPE_INTEGER) {
+          if ($1.value.integer == 0) {
+            yywarning(yyscanner,
+                "Consider using \"none\" keyword, it is less ambiguous. Please "
```

Comment:
Start errors and warning with lowercase characters. 

Also, even though the link may be useful, I'm not sure about including it. This warning will be shown in contexts in which a long plain-text link may be cause issues. I'm thinking about VT's web interface for example. So, I would keep the link out of the message.